ml_service/1_topic_modeling/fetch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+DATADIR=$(dirname "$0")
+DATASET=20news-18828
+
+aws s3 sync s3://squirro-datasets/walkthroughs/$DATASET $DATADIR --profile squirro-internal
```

Comment:
This repo is intended mainly for partners, so I wouldn't count on anyone being able to access our squirro-internal AWS buckets. 
I'm going to put together a separate PR that re-works this across the board to make it easier

Reply:
across the board?